vale-linting/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Vale linting
+description: Vale is a command-line tool that brings code-like linting to prose.
+inputs:
+  exclude:
+    description: |
+      A file or files that should be excluded from linting.
+      New line separated string with micromatch patterns to exclude files.
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Lint natural language with Vale
+      # Change to the latest version/tag
+      uses: l0kal/vale-action@master
```

Comment:
Is this a fork of the official vale-action? If this is long-term, I think we should fork in the extenda organization instead.